x-pack/plugins/apm/public/utils/formatters.js

```diff
@@ -52,3 +52,14 @@ export function asInteger(value) {
 export function tpmUnit(type) {
```

Comment:
Can we convert this file to `.ts` to preserve the typings on that function? (Does it not blow up like this? I'm not sure what happens with TS in a .js file...)

Reply:
Agree, would be nice to get this file converted. If you do, please remove all the `// @ts-ignore`, eg:
https://github.com/elastic/kibana/blob/81b334b28625f8a40fad262d4fb12008694083f8/x-pack/plugins/apm/public/components/app/TraceOverview/TraceList.tsx#L12-L13

Importing .ts in a js file should be fine